apps/widhwbttm/widhwbttm.wid.js

```diff
@@ -1,7 +1,8 @@
 (function() {
+  let intervalRef = null;
     var v_switch; // show stats
-    if (process.env.BOARD=='BANGLEJS'||process.env.BOARD=='EMSCRIPTEN') v_font_size=16
-     else  v_font_size=13;
+    if (process.env.BOARD=='BANGLEJS'||process.env.BOARD=='EMSCRIPTEN') v_font_size=15
```

Comment:
It looks like v_font_size is defined as a global variable here? Please could you define it locally with `var`?

Reply:
Sure, thanx